R/zchunk_L203.demand_input.R

```diff
@@ -37,9 +37,9 @@ module_aglu_L203.demand_input_DISABLED <- function(command, ...) {
              "L101.Pop_thous_R_Yh",
              "L102.pcgdp_thous90USD_Scen_R_Y"))
   } else if(command == driver.DECLARE_OUTPUTS) {
-    return(c("L203.SectorLogitTables[[ curr_table ]]$data",
+    return(c("L203.SectorLogitTables[[curr_table]]",
```

Comment:
Error message like:
Failed -------------------------------------------------------------------------
1. Failure: matches old data system output (@test_oldnew.R#66) -----------------
length(oldf) == 1 isn't true.
Either zero, or multiple, comparison datasets found for L203.SectorLogitTables[[curr_table]]$data.csv

2. Failure: matches old data system output (@test_oldnew.R#66) -----------------
length(oldf) == 1 isn't true.
Either zero, or multiple, comparison datasets found for L203.SubsectorLogitTables[[curr_table]]$data.csv